RNA-seq/setup/04-nb_cell_line_tximport.Rmd

```diff
@@ -0,0 +1,203 @@
+---
+title: "NB cell line: gene-level summarization with `tximport`"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2018**
+
+This is the solved notebook version as a result of the prompt in `04-nb_cell_line_tximport.md`.
```

Comment:
This notebook is basically exactly the same as `02-gastric_cancer_tximport.Rmd` except for this line and I search and replaced `gastric-cancer` with `NB-cell`.

Reply:
and some added `..` in file paths